deploy/helm/cf-operator/templates/fissile_v1alpha1_extendedsecret_crd.yaml

```diff
@@ -18,4 +18,6 @@ spec:
         - esecs
   scope: Namespaced
   version: v1alpha1
+  subresources:
```

Comment:
CRD in code should be synced: https://github.com/cloudfoundry-incubator/cf-operator/blob/245486dcde519f54ca4a27ec92c1ee6443eb1247/pkg/kube/util/crd/crd.go#L18-L38

Reply:
We'll move it to docs soon enough ;)

I think it's in code already: https://github.com/cloudfoundry-incubator/cf-operator/pull/459/files#diff-0c1a28ec9d4ea775ba31f02b722af57dR31